### PR TITLE
[ENH] Retrieve labels and parcellation image from PairwiseAlignment

### PR DESCRIPTION
**Pull request by pbarbarant**
Adds a public method to PairwiseAlignment API to retrieve both the list of labels and the parcellation as a single Niftiimage.

- [x] Docstrings
- [x] Unit tests

### CODE REVIEW

File: fmralign/pairwise_alignment.py [hunk 1]
```diff
@@ -274,3 +275,26 @@ def fit_transform(self):
...
+        Returns
+        -------
+        labels: `list` of `int`
+            Labels of the parcellation masker.
+        parcellation_img: Niimg-like object
+            Parcellation image.
+        """
+        if hasattr(self, "pmasker"):
+            check_is_fitted(self)
+            labels = self.pmasker.get_labels()
```
Comment (bthirion) [inline fmralign/pairwise_alignment.py:291]: I really hat the `pmasker`. Can you change it to `parcel_masker` ?

**Review by bthirion — COMMENTED** [review]
LGTM overall.

File: fmralign/pairwise_alignment.py (hunk 1 above)
Comment (pbarbarant) [reply fmralign/pairwise_alignment.py]: Sure!

**Comment by pbarbarant** [comment]
Closes #109
